src/main/java/ab/common/lib/register/RecipeListAB.java

```diff
@@ -405,9 +413,8 @@ public static void setupCrafting() {
                 new ItemStack(BlockListAB.blockManaCrystalCube),
                 new Object[] { "TST", "GWG", "DMD", Character.valueOf('D'), new ItemStack(ModBlocks.dreamwood),
                         Character.valueOf('W'), new ItemStack(ModItems.twigWand, 1, 32767), Character.valueOf('G'),
-                        new ItemStack(ModBlocks.manaGlass), Character.valueOf('M'),
-                        new ItemStack(ModItems.manaResource), Character.valueOf('T'),
-                        new ItemStack(ModItems.manaResource, 1, 18), Character.valueOf('S'),
+                        new ItemStack(ModBlocks.manaGlass), Character.valueOf('M'), new ItemStack(ModBlocks.storage),
+                        Character.valueOf('T'), new ItemStack(ModItems.manaResource, 1, 18), Character.valueOf('S'),
```

Comment:
This looks a lot like symptomatic decompiled code and it makes things unnecessarily hard to read. While I don't expect you to fix the entire codebase at once, I would recommend that changed recipes follow idiomatic java. Compare with:

```
GameRegistry.addShapedOreRecipe(
  new ItemStack(BlockListAB.blockManaCrystalCube),
  "TST", "GWG", "DMD",
  'D', new ItemStack(ModBlocks.dreamwood),
  'W', new ItemStack(ModItems.twigWand, 1, 32767), // is the magic number necessary?
  'G', new ItemStack(ModBlocks.manaGlass),
  'M', "blockManaSteel", // not sure if its the right one, but must be oredict for GT compat
  'T', new ItemStack(ModItems.manaResource, 1, 18), // prefer oredict
  'S', new ItemStack(ModItems.spark)
);
```

Reply:
32767 == `OreDictionary.WILDCARD_VALUE`

Reply:
@combusterf I need to change this? I mean if I make new recipes ( that for the time I dont think so), I will use this method, also the magic number is because the recipe can use every single wand of the forest